onetl/db/db_reader/db_reader.py

```diff
@@ -398,6 +398,9 @@ def validate_hwm_column(cls, values: dict) -> dict:
         values["hwm_column"] = Column(name=hwm_column)  # type: ignore
         values["hwm_expression"] = hwm_expression
 
+        dialect = connection.Dialect
+        dialect.validate_hwm_column(connection, hwm_column)  # type: ignore
```

Comment:
```suggestion
        dialect = connection.Dialect
        dialect.validate_hwm_column(connection, hwm_column)  # type: ignore

        values["hwm_column"] = Column(name=hwm_column)  # type: ignore
        values["hwm_expression"] = hwm_expression
```
я бы сделал так, а то странно проводить валидацию после присвоения в values